odata-url-conventions/4 Resource Path.md

```diff
@@ -985,10 +985,12 @@ Requests to paths ending in `/$query` MUST use the `POST` verb. Query
 options specified in the request body and query options specified in the
 request URL are processed together.
 
-The request body MUST use `Content-Type: text/plain`. It contains the
-query portion of the URL and MUST use the same percent-encoding as in
-URLs (especially: no spaces, tabs, or line breaks allowed) and MUST
-follow the syntax rules described in chapter Query Options.
+The request body MUST use `Content-Type: text/plain` or `Content-Type: application/x-www-form-urlencoded`.
+It contains the query portion of the URL.
+
+For `Content-Type: text/plain`, the individual query options MUST be separated by `&` or newlines
+and MUST use the same percent-encoding as in URLs (especially: no spaces, tabs, or line breaks allowed)
+and MUST follow the syntax rules described in [chapter ##QueryOptions].
```

Comment:
This does not follow the normative [`text/plain` encoding algorithm](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#text/plain-encoding-algorithm), which allows only separation by newlines.

Reply:
Do you want to open an issue for only allowing newline?

Reply:
@mikepizzo, your opinion is appreciated.